### DISCUSSION TITLE
Is it possible to disable the generated "-mask-composite" files?

### DISCUSSION THREAD

**Discussion by onyxlee** [Bing-su/adetailer#150] [Q&A]
Hi, dear community, I am pretty satisfied with adetailer, so I feel it's no longer necessary for me to save the face part of the PNG seperately. Is it possible to disable the generated "-mask-composite" files? I checked the settings, it doesn't seem like any settings in that section have any effects. The option "save meask previews" is always disabled, but it still generates that face PNG.

What's your opinion on this? Thanks.

**Comment by Bing-su** [comment]
adetailer only uses `-ad-before` and `-ad-preview`. `-mask-composite` must have come from your other settings.
